CONTRIBUTING.md

```diff
@@ -67,3 +67,6 @@ curl --location --silent --show-error \
 ```
 
 Note, for this example to work, you would need to have `kubectl` installed and configured to talk to a Kubernetes cluster.
+
+## Updating your application
+* test by running deploy-local.sh

```

Comment:
Any integration testing we should call out, and ask the user to provide? I also have a PR coming that will automatically do a deploy on a PR>

Reply:
Good call out. Until we have that in place i think we shouldn’t mention it. When your pr goes in we can add a “testing spot”. It would be nice to have a clear sense of what testing on this repo looks like.

Reply:
Good call out. Until we have that in place i think we shouldn’t mention it. When your pr goes in we can add a “testing spot”. It would be nice to have a clear sense of what testing on this repo looks like.

Reply:
Good call out. Until we have that in place i think we shouldn’t mention it. When your pr goes in we can add a “testing spot”. It would be nice to have a clear sense of what testing on this repo looks like.

Reply:
Good call out. Until we have that in place i think we shouldn’t mention it. When your pr goes in we can add a “testing spot”. It would be nice to have a clear sense of what testing on this repo looks like.

Reply:
Good call out. Until we have that in place i think we shouldn’t mention it. When your pr goes in we can add a “testing spot”. It would be nice to have a clear sense of what testing on this repo looks like.

Reply:
Good call out. Until we have that in place i think we shouldn’t mention it. When your pr goes in we can add a “testing spot”. It would be nice to have a clear sense of what testing on this repo looks like.

Reply:
Good call out. Until we have that in place i think we shouldn’t mention it. When your pr goes in we can add a “testing spot”. It would be nice to have a clear sense of what testing on this repo looks like.

Reply:
Good call out. Until we have that in place i think we shouldn’t mention it. When your pr goes in we can add a “testing spot”. It would be nice to have a clear sense of what testing on this repo looks like.